legend-depot-core-services/src/main/java/org/finos/legend/depot/services/projects/ProjectsServiceImpl.java

```diff
@@ -228,7 +228,7 @@ public Set<ProjectVersion> getDependencies(List<ProjectVersion> projectVersions,
             {
                 if (projectData.getTransitiveDependenciesReport().isValid())
                 {
-                    dependencies.addAll(projectData.getTransitiveDependenciesReport().getTransitiveDependencies());
+                    dependencies.addAll(overrideDependencies(projectVersions, projectData.getTransitiveDependenciesReport().getTransitiveDependencies()));
```

Comment:
we should not chnage the client just the logic..and do it once as it will never change